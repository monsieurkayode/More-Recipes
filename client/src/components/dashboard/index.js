/* jshint esversion: 6 */
import UserProfile from './UserProfile';
import UserRecipe from './UserRecipe';
import WelcomeDisplay from './WelcomeDisplay';
import DashboardPanel from './DashboardPanel';
import SideNavDashboard from './SideNavDashboard';
import UserFavoriteRecipe from './UserFavoriteRecipe';
import UserNotification from './UserNotification';

export {
  UserProfile,
  UserRecipe,
  UserFavoriteRecipe,
  UserNotification,
  WelcomeDisplay,
  SideNavDashboard,
  DashboardPanel
};
