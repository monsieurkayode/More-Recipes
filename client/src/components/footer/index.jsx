import React from 'react';

/**
 * Footer
 * @function Footer
 *
 * @param {void} void
 *
 * @return {JSX} JSX
 */
const Footer = () => (
  <footer className="page-footer blue-grey darken-2">
    <div className="row">
      <div className="col l3 social-icon-container">
        <ul>
          <li>
            <div>
              <span>
                <i className="fa fa-2x fa-github teal-text" />
              </span>
            </div>
          </li>
          <li>
            <div>
              <span>
                <i className="fa fa-2x fa-facebook teal-text" />
              </span>
            </div>
          </li>
          <li>
            <div>
              <span>
                <i className="fa fa-2x fa-twitter teal-text" />
              </span>
            </div>
          </li>
          <li>
            <div>
              <span>
                <i className="fa fa-2x fa-google teal-text" />
              </span>
            </div>
          </li>
        </ul>
      </div>
    </div>
    <div className="footer-copyright">
      <div className="container">
        Â© Andela 2018
        <a
          className="grey-text text-lighten-4 right"
          href="/"
        >
        Created by Kayode Adeola
        </a>
      </div>
    </div>
  </footer>
);

export default Footer;
