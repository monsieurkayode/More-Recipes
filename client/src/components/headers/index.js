/* jshint esversion: 6 */
import Banner from './Banner';
import DashboardNavbar from './DashboardNavbar';
import HomeNavbar from './HomeNavbar';
import LandingNavbar from './LandingNavbar';
import UserMenu from './UserMenu';
import SearchBar from './SearchBar';

export {
  Banner,
  DashboardNavbar,
  HomeNavbar,
  LandingNavbar,
  UserMenu,
  SearchBar
};
