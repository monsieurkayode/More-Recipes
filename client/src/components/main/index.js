/* jshint esversion: 6 */
import Category from './Category';
import Recipes from './Recipes';
import RecipeItem from './RecipeItem';
import TopRecipeItem from './TopRecipeItem';
import TopRecipes from './TopRecipes';
import SignupForm from './SignupForm';
import SigninForm from './SigninForm';
import TextField from './TextField';
import PostRecipe from './PostRecipe';
import EditRecipe from './EditRecipe';
import SideNav from './SideNav';
import WelcomeMessage from './WelcomeMessage';
import Loader from './Loader';

export {
  Category,
  Recipes,
  RecipeItem,
  TopRecipeItem,
  TopRecipes,
  PostRecipe,
  EditRecipe,
  SignupForm,
  SigninForm,
  TextField,
  SideNav,
  WelcomeMessage,
  Loader
};
