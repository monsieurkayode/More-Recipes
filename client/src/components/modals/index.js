/* jshint esversion: 6 */
import DeleteModal from './DeleteModal';

export default DeleteModal;
