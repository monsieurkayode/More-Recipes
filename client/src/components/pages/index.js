/* jshint esversion: 6 */
import SigninPage from './SigninPage';
import SignupPage from './SignupPage';
import DashboardPage from './DashboardPage';
import NotFoundPage from './NotFoundPage';
import RecipeViewPage from './RecipeViewPage';
import PostRecipePage from './PostRecipePage';
import IndexPage from './IndexPage';
import EditRecipePage from './EditRecipePage';
import HomePage from './Homepage';
import CategoryPage from './CategoryPage';

export {
  SigninPage,
  SignupPage,
  DashboardPage,
  NotFoundPage,
  RecipeViewPage,
  PostRecipePage,
  IndexPage,
  EditRecipePage,
  HomePage,
  CategoryPage
};
