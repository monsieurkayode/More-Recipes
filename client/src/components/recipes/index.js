import RecipeCard from './RecipeCard';
import RecipeCardSmall from './RecipeCardSmall';

export {
  RecipeCard,
  RecipeCardSmall
};
