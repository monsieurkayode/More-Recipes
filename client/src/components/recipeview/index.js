import Comments from './Comments';
import Ingredients from './Ingredients';
import Instructions from './Instructions';
import CommentBox from './CommentBox';

export {
  Comments,
  Ingredients,
  Instructions,
  CommentBox
};
