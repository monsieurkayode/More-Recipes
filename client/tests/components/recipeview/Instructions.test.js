/* eslint-disable no-undef */
import React from 'react';

import { Instructions } from '../../../src/components/recipeview';
import data from '../../__mocks__/mockData';

const setup = () => {
  const props = {
    ingredients: data.recipeDetails.instructions
  };

  const wrapper = shallow(<Instructions {...props} />);

  return {
    wrapper
  };
};

const { wrapper } = setup();

describe('Instructions', () => {
  it('should render correctly',
    (done) => {
      expect(toJson(wrapper)).toMatchSnapshot();
      done();
    });
});
