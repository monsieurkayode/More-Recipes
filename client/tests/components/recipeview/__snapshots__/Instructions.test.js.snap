// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Instructions should render correctly 1`] = `
<div
  className="col l12 m12 s12"
>
  <h4>
    Instructions
  </h4>
  <div
    dangerouslySetInnerHTML={
      Object {
        "__html": "",
      }
    }
    id="instructions"
  />
</div>
`;
